aws/rust-runtime/aws-config/src/environment/timeout_config.rs

```diff
@@ -5,25 +5,27 @@
 
 //! Load timeout configuration properties from environment variables
 
-use aws_smithy_types::timeout::{parse_str_as_timeout, TimeoutConfig, TimeoutConfigError};
+use aws_smithy_types::timeout::{self, parse_str_as_timeout};
+use aws_smithy_types::tristate::TriState;
 use aws_types::os_shim_internal::Env;
 use std::time::Duration;
 
+// Currently unsupported timeouts
 const ENV_VAR_CONNECT_TIMEOUT: &str = "AWS_CONNECT_TIMEOUT";
 const ENV_VAR_TLS_NEGOTIATION_TIMEOUT: &str = "AWS_TLS_NEGOTIATION_TIMEOUT";
 const ENV_VAR_READ_TIMEOUT: &str = "AWS_READ_TIMEOUT";
+
+// Supported timeouts
 const ENV_VAR_API_CALL_ATTEMPT_TIMEOUT: &str = "AWS_API_CALL_ATTEMPT_TIMEOUT";
 const ENV_VAR_API_CALL_TIMEOUT: &str = "AWS_API_CALL_TIMEOUT";
 
 /// Load a timeout_config from environment variables
 ///
-/// This provider will check the values of the following variables in order to build a `TimeoutConfig`
+/// This provider will check the values of the following variables in order to build a
+/// [`timeout::Config`](aws_smithy_types::timeout::Config)
 ///
-/// - `AWS_CONNECT_TIMEOUT`
-/// - `AWS_TLS_NEGOTIATION_TIMEOUT`
-/// - `AWS_READ_TIMEOUT`
-/// - `AWS_API_CALL_ATTEMPT_TIMEOUT`
-/// - `AWS_API_CALL_TIMEOUT`
+/// - `ENV_VAR_API_CALL_ATTEMPT_TIMEOUT`
+/// - `ENV_VAR_API_CALL_TIMEOUT`
```

Comment:
For the docs, shouldn't these be the actual environment variable names rather than the names of the constants?

Reply:
ah, yeah, you're right. I should have left it